tools/layer-zero-example/contracts/hts/HederaResponseCodes.sol

```diff
@@ -0,0 +1,330 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.4.9 <0.9.0;
+
+library HederaResponseCodes {
```

Comment:
nit: might not be necessary to dupe the whole file. 
Why not just pull in the response code you need and add a comment ref to the HederaResponseCodes in the smart contracts repo.
Once we get the npm package this will be easier

Reply:
Done.